### DISCUSSION TITLE
Tipo de impuesto en grupo de impuesto

### DISCUSSION THREAD

**Discussion by FDMilesi** [OdooAR/odoo-argentina-doc#19] [General]
Quizás ya es un problema conocido, pero hay que configurar a mano el tax_type en el modelo account.tax.group, para IVA. Sino genera mal la xml request para afip.

**Comment by ibuioli** [comment]
Es cierto, en algunas localizaciones pasa eso. En la versión que estamos armando para OCA lo sacamos. Algo que se puede hacer para las demás localizaciones es quizás un XML en data que actualice automáticamente los impuestos.
